.cubic/wiki/03-section-features/05-p-project-workspace.md

```
@@ -116,6 +117,15 @@ export const normalizeStoredProject = (data: unknown): ProjectSnapshot => {
 
 Sources: [apps/web/services/projects/projectSnapshot.ts:600-630](../../../apps/web/services/projects/projectSnapshot.ts#L600-L630)
 
+### Archive Import Identity Remapping
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)